united-states-international-trade-commission.csl

```diff
@@ -323,6 +340,27 @@
               </else>
             </choose>
           </else-if>
+          <else-if type="software" match="any">
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title-short" form="short" text-case="title"/>
+              </if>
+              <else>
+                <text variable="title" form="short" text-case="title"/>
+              </else>
+            </choose>
+          </else-if>
+          <else-if type="speech" match="any"/>
+          <else-if type="hearing" match="any">
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title-short"/>
+              </if>
+              <else>
+                <text variable="title"/>
+              </else>
+            </choose>
```

Comment:
```suggestion
            <text variable="title" form="short"/>
```
does the exact same thing as this whole test. ditto below -- please change accordingly